MEMBER_EXPECTATIONS.md

```diff
@@ -1,31 +1,58 @@
+All participants in OpenJS Foundation projects and Working Groups must follow the [Code of Conduct](https://github.com/openjs-foundation/cross-project-council/blob/HEAD/CODE_OF_CONDUCT.md). Standards Working Group Members are expected to participate in discussions related to OpenJS Foundation Projects, open source software, and the standards and specifications that bear on them. They are also expected educate and mentor newcomers to the field of standardization, lending support and advice to projects pursuing specification development. Finally, they are expected to help export OpenJS Foundation values to the standards communities they work with in order to make standards bodies nicer places to do work.  
 
-It is understood that representatives may form opinions based on the needs of their employers' or their personal projects. While representing
-the Standards Working Group, however, they should decide/discuss/promote based on the interest of the Standards Working Group.
-If there is a conflict (due to personal or company specific projects), then the representative should be explicit
-about the hat they are wearing while deciding/discussing/promoting the project.
+There are further expectations for [OpenJS Foundation Representatives](https://github.com/openjs-foundation/standards/blob/main/MEMBER_REPRESENTATION.md) to standards and liaison organizations (hereafter called Representatives).
 
-Since many members are often a part of one or more projects in the foundation, feel free to discuss it with the members
-if there is a potential conflict while representing. The representative can raise an issue and discuss them during
-the Standards Working Group meeting.
+## Expectations for Representatives
+It is understood that Representatives may form opinions based on the needs of their employers, personal projects, or other experiences. While representing the Foundation, however, Representatives should participate based on what is in the best interest of the Foundation and its projects. If there is a conflict (e.g. between the Foundation, its projects, and/or personal or company positions), the Representative should be explicit about which perspective they are speaking from given the issue at hand. 
 
-If the representative or any member of Standards Working Group feels there will be a controversial topic, they can call for a 
-`pre-meeting` to discuss it with the other members of the Working Group. Our representatives should always need to focus on
-representing the foundation and the consensus/expectations that we’ve built to those organizations (if they are representing 
-OpenJS Foundation). 
+Representatives must also conduct themselves in a professional and respectful manner. General guidelines include:
+* Avoid invoking multiple "roles" when discussing challenging topics. This helps avoid [appeals to authority](https://en.wikipedia.org/wiki/Argument_from_authority) or the allusion that one's opinion is fully shared by multiple groups. 
```

Comment:
This addresses the importance of avoiding "pulling rank" and misrepresenting others' positions.  You're right that there's nothing wrong with "documented consensus that you can point to" and this can help clarify things and/or prevent bikeshedding.